e2e/testcases/reconciler_finalizer_test.go

```diff
@@ -116,16 +117,14 @@ func TestReconcilerFinalizer_Orphan(t *testing.T) {
 		// Namespace1 should NOT have been deleted, because it was orphaned by the RootSync.
 		return nt.Watcher.WatchObject(kinds.Namespace(), namespace1.GetName(), namespace1.GetNamespace(),
 			testwatcher.WatchPredicates(
-				testpredicates.HasAllNomosMetadata(), // metadata NOT removed when orphaned
-				testpredicates.HasLabel(metadata.ApplySetPartOfLabel, applySetID),
+				testpredicates.IsNotManaged(nt.Scheme),
```

Comment:
Does this validate just the managed annotation or all the removed metadata?

Reply:
It [validates the managed annotation](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/nomostest/testpredicates/predicates.go#L674)

Reply:
Looks like both the manager and managed annotations.

If we're gonna remove all the CS meta, we should also validate all of it was removed.